usagov/vote-gov-drupal
# Vote-1045:  Dependabot updates

## Jira ticket

[Vote-1045](https://cm-jira.usa.gov/browse/VOTE-1045)

## Description

Updates from Dependabot 

Closes out:

- https://github.com/usagov/vote-gov-drupal/pull/617
- https://github.com/usagov/vote-gov-drupal/pull/637
- https://github.com/usagov/vote-gov-drupal/pull/638
- https://github.com/usagov/vote-gov-drupal/pull/647

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. cd into `testing` and run `npm i` ensure there are no build issues
2. start local and run `npm run cy:fullTests` and ensure tests are passing 
3. cd into the `votegov` theme and run `npm i` and ensure there are no build issues 
4. run `npm run build` and `npm run gulp` and check no errors in build or run

## Checklist for the Developer




## Checklist for the Peer Reviewers
